lib/buffer.js

```diff
@@ -237,6 +237,17 @@ Buffer.from = function from(value, encodingOrOffset, length) {
   );
 };
 
+// Identical to the built-in %TypedArray%.of(), but avoids using the deprecated
+// Buffer() constructor.
+//
+// Refs: https://tc39.github.io/ecma262/#sec-%typedarray%.of
+Buffer.of = (...items) => {
```

Comment:
This changes the name of the function. It should probably be named `of`.

Reply:
Grrr, I always thought this kind of assignment names the function automatically.

Reply:
They are named in cases like `{ of: () => {} }` and `const of = () => {}`, but not in `x.of = () => {}`.